04_sessions.html

```diff
@@ -109,89 +109,36 @@
         <td></td>
       </tr>
     </table>
-    <p>The process for successfully creating a session MUST be:</p>
+    <p>The <a>remote end steps</a> for the <dfn>New Session</dfn> command are:</p>
```

Comment:
There should probably be a note here about intermediate nodes, or something, so that someone doesn't complain that this isn't how they work when they're distributing jobs. Basically the intermediate node can do anything it likes with the capabilities and typically just forwards the session id from the furthest remote.